### PR TITLE
Update links for repository moves/renames

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,10 +16,10 @@ Further development of the EF6 PowerTools is happening in a community-driven pro
...
+If you want to try out the latest changes that have not been officially released yet, you can choose to [build the code](https://github.com/dotnet/ef6/wiki/Building-the-Runtime). We regularily also make [nightly builds](https://github.com/dotnet/ef6/wiki/Nightly-Builds) of the Entity Framework codebase available.
 
 ## How do I contribute
 
-There are lots of ways to [contribute to the Entity Framework project](https://github.com/aspnet/EntityFramework6/wiki/Contributing) including [testing out nighty builds](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
+There are lots of ways to [contribute to the Entity Framework project](https://github.com/dotnet/ef6/wiki/Contributing) including [testing out nighty builds](https://github.com//blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
```
Comment (lajones) [inline README.md]: The replacement for aspnet/AspNetCore looks like it's missing?

Comment (Pilchie) [reply README.md]:
```suggestion
There are lots of ways to [contribute to the Entity Framework project](https://github.com/dotnet/ef6/wiki/Contributing) including [testing out nighty builds](https://github.com/dotnet/aspnetcore/blob/master/docs/DailyBuilds.md), reporting bugs, and contributing code.
```

**Comment by lajones** [comment]
Looks good to me.

**Comment by Pilchie** [comment]
@lajones - does the test failure above look like flakiness, or could it be a real issue?

**Comment by lajones** [comment]
@Pilchie Looks like flakiness to me. I'd just kick them off again and see what happens.

**Pull request merged**